packages/plugin-ext/src/plugin/tasks/tasks.ts

```diff
@@ -49,9 +52,9 @@ export class TasksExtImpl implements TasksExt {
 
     constructor(rpc: RPCProtocol, readonly terminalExt: TerminalServiceExtImpl) {
         this.proxy = rpc.getProxy(PLUGIN_RPC_CONTEXT.TASKS_MAIN);
-        this.providedCustomExecutions = new Map<number, CustomExecution>();
-        this.notProvidedCustomExecutions = new Set<number>();
-        this.activeCustomExecutions = new Map<number, CustomExecution>();
+        this.customExecutionIds = new Map();
```

Comment:
[Minor]: some map fields are initialized as properties above (`adaptersMap`, `executions`); others are initialized here. I'd vote for moving these three, which don't need information from the constructor arguments, up with the rest and initializing with the property syntax.